cli/tests/error_local_static_import_from_remote.js.out

```diff
@@ -1,3 +1,4 @@
 [WILDCARD]
-error: Remote modules are not allowed to statically import local modules. Use dynamic import instead.
-Imported from "[WILDCARD]error_local_static_import_from_remote.js:1"
+error: Remote modules are not allowed to import local modules.
+  Importing: file:///some/dir/file.js
```

Comment:
Sad to see `Use dynamic import instead` hint go away

Reply:
I will fix that.